wekan/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: wekan
 sources:
   - https://github.com/wekan/wekan
 type: application
-version: 7.01.0
+version: 7.01.1
```

Comment:
Remove this change, version numbering does not go like that.

Reply:
Done, I removed the commit.